monero-outreach-docs/translations/es/PSA/files/teamwork/teamwork_es.sla

```diff
@@ -64,15 +67,44 @@
+        <PAGEOBJECT OwnPage="0" PTYPE="4" XPOS="136" YPOS="380" WIDTH="255.6" HEIGHT="287.0016" RADRECT="0" FRTYPE="0" CLIPEDIT="0" PWIDTH="1" PCOLOR="None" PCOLOR2="None" TXTFILLSH="90" COLUMNS="1" COLGAP="0" NAMEDLST="" SHADE="100" SHADE2="100" GRTYP="0" ROT="0" PLINEART="1" PLINEEND="0" PLINEJOIN="0" LINESP="13.2" LOCALSCX="1" LOCALSCY="1" LOCALX="0" LOCALY="0" PICART="1" PLTSHOW="0" BASEOF="0" textPathType="0" textPathFlipped="0" FLIPPEDH="0" FLIPPEDV="0" IFONT="Adobe Caslon Pro Regular" ISIZE="11" SCALETYPE="1" RATIO="1" PRINTABLE="1" ANNOTATION="0" ANNAME="" TEXTFLOWMODE="0" TEXTFLOW="0" TEXTFLOW2="0" TEXTFLOW3="0" AUTOTEXT="0" EXTRA="0" TEXTRA="0" BEXTRA="0" REXTRA="0" FLOP="0" PFILE="" PFILE2="" PFILE3="" PRFILE="" EPROF="" IRENDER="1" EMBEDDED="1" LOCK="1" LOCKR="0" REVERS="0" TransValue="0" TransValueS="0" TransBlend="0" TransBlendS="0" isTableItem="0" TopLine="0" LeftLine="0" RightLine="0" BottomLine="0" isGroupControl="0" NUMDASH="0" DASHS="" DASHOFF="0" NUMPO="16" POCOOR="0 0 0 0 255.6 0 255.6 0 255.6 0 255.6 0 255.6 287.002 255.6 287.002 255.6 287.002 255.6 287.002 0 287.002 0 287.002 0 287.002 0 287.002 0 0 0 0 " NUMCO="16" COCOOR="0 0 0 0 255.6 0 255.6 0 255.6 0 255.6 0 255.6 287.002 255.6 287.002 255.6 287.002 255.6 287.002 0 287.002 0 287.002 0 287.002 0 287.002 0 0 0 0 " NUMGROUP="0" GROUPS="" startArrowIndex="0" endArrowIndex="0" OnMasterPage="" ImageClip="" ImageRes="1" Pagenumber="0" isInline="0" fillRule="1" doOverprint="0" gXpos="0" gYpos="0" gWidth="0" gHeight="0" LAYER="0" BOOKMARK="0" NEXTITEM="-1" BACKITEM="-1">
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="Monero es un proyecto de código abierto completamente descentralizado. Cualquiera con una buena idea e iniciativa puede crear y liderar un proyecto que apoye a Monero. Si no eres un programador y estas interesado "/>
+            <para LINESP="13.2"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="en soportar a Monero en otras formas, existen muchas opciones. Los proyectos de Monero se organizan en varios equipos de trabajo. Hay un equipo de trabajo conocido como Localizador enfocado en traducir todo "/>
```

Comment:
"en **apoyar** a Monero **de** otras formas...".

Reply:
"Hay un equipo de trabajo conocido como **Localizador** enfocado en traducir todo". The word comes from "localization" and it have sense in English, but in the case of translating it to Spanish it loses its meaning (localización). I suggest a discussion about this and about what can we use in Spanish for that word and their respective conjugations. We can jut simply leave it in English, or we can use another word with the respective meaning like _traducir_.

Reply:
I used it as "localizador" of languages. But you are right. I will change it to **"localization"** until we find a better word for it.

Reply:
done